Userland/Libraries/LibSQL/Result.h

```diff
@@ -63,7 +63,8 @@ constexpr char const* command_tag(SQLCommand command)
     S(BooleanOperatorTypeMismatch, "Cannot apply '{}' operator to non-boolean operands")          \
     S(NumericOperatorTypeMismatch, "Cannot apply '{}' operator to non-numeric operands")          \
     S(IntegerOperatorTypeMismatch, "Cannot apply '{}' operator to non-numeric operands")          \
-    S(InvalidOperator, "Invalid operator '{}'")
+    S(InvalidOperator, "Invalid operator '{}'")                                                   \
+    S(IntegerOverflow, "Operation would cause integer overflow")
```

Comment:
:red_circle: :penguin:

Reply:
Oh nvm the rest isn't sorted either

Reply:
Yeah this did feel dirty though so I'll sort them too

Reply:
*SORTED*